slices/libnss-mdns.yaml

```diff
@@ -0,0 +1,51 @@
+package: libnss-mdns
+
+essential:
+  - libnss-mdns_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libnss-mdns_full
+      - libnss-mdns_minimal
+
+  full:
```

Comment:
is `full` the right name? Did you come up with it or is it how the package references these libs?

Reply:
The package doesn't reference those libs individually, but the lib file names indicate whether they are minimal or otherwise (aka full), and the version numbers, so it looks that this slice naming is reasonable.